lib/index.js

```diff
@@ -10,20 +10,24 @@ class Modem {
       messageDelay: 1,
       bigMessageDelay: 30,
       sendMessageStatusDelay: 1,
-      waitForPendingRequest: true
+      waitForPendingRequest: true,
+      webUiVersion: undefined,
     }
 
     this.modemIp = options.modemIp || this.defaultOptions.modemIp
     this.messageDelay = options.messageDelay || this.defaultOptions.messageDelay
     this.bigMessageDelay = options.bigMessageDelay || this.defaultOptions.bigMessageDelay
     this.sendMessageStatusDelay = options.sendMessageStatusDelay || this.defaultOptions.sendMessageStatusDelay
     this.waitForPendingRequest = options.waitForPendingRequest || this.defaultOptions.waitForPendingRequest
+    this.webUiVersion = this.defaultOptions.webUiVersion ? this.defaultOptions.webUiVersion : undefined;
```

Comment:
Change this line to `this.webUiVersion = options.webUiVersion || this.defaultOptions.webUiVersion`